android/src/ti/airship/Utils.java

```diff
@@ -0,0 +1,14 @@
+package ti.airship;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public abstract class Utils {
+    public static Set<String> convertToStringSet(Object[] values) {
```

Comment:
Should we be checking the types here or elsewhere? While valueOf should always return a string we could end up with weird contents if other kinds of objects were passed in for some reason. Might make sense to skip and log in that case.